tenable_io/api/models.py

```diff
@@ -2182,6 +2276,118 @@ def assets(self, assets):
         self._assets = assets
 
 
+class AssetsAssetSource(BaseModel):
+
+    def __init__(
+            self,
+            name=None,
+            last_seen=None,
+            first_seen=None
+    ):
+        self.name = name
+        self.last_seen = last_seen
+        self.first_seen = first_seen
+
+
+class AssetsAsset(BaseModel):
```

Comment:
I think once the public api documentation is updated we'll be able to use the same Asset model for this and the workbenches asset results. However, having two models is fine for now.